desc/objectives/_bootstrap.py

```diff
@@ -85,6 +99,7 @@ def __init__(
         grid=None,
         helicity=(1, 0),
         name="Bootstrap current self-consistency (Redl)",
+        jac_chunk_size="auto",
```

Comment:
for the `_Objective`s I think `None` would be a better default (consistent with current behavior), and remove the "auto" option since it's just hard-coded to 1000.